src/Widgets/PageItem.vala

```diff
@@ -77,6 +92,15 @@ public class ENotes.PageItem : Gtk.ListBoxRow {
     }
 
     public void load_data () {
+        time = new DateTime.from_unix_utc (page.modification_date);
+
+        if (use24HSFormat) {
+            date_string = time.format ("%H:%M, %a, %e %b %Y");
```

Comment:
- You should use the same format that you did on your mockup :) 
    - EG: `Saturday, May 17, 2018, 9:26`
- Make the string translatable: surround the string within `_("")`